sample/HttpTrigger-TypeScript/index.ts

```diff
@@ -0,0 +1,29 @@
+﻿export function index(context: any, req: any) {
```

Comment:
Even without those, we should be able to enhance this sample quite a bit. It's pretty plain right now.